content/webapp/components/EventSchedule/EventScheduleItem.tsx

```diff
@@ -98,30 +99,58 @@ const eventLocations = (locations: Place[], isHybridEvent: boolean) => {
 
 // We have a message block on scheduled events which either displays
 // 'Just turn up' or 'Arrive early to register'
-// We only show this message if:
-// - the event isn't past AND
-// - it doesn't require booking (either through Eventbrite or by contacting the booking enquiry team) AND
-// - it doesn't have it's own event schedule AND
-// - the parent event isn't Ticketed
-function shouldShowMessage({
-  event,
-  parentEvent,
-}: {
-  event: Event;
-  parentEvent: Event;
-}): boolean {
+// N.B. If the criteria to display both messages is satisfied, we only show 'Arrive early to register' (see below)
+// We don't show either message if the following criteria aren't satisfied:
+function shouldShowMessage(event: Event): boolean {
   return (
     !isEventPast(event) &&
     !event.eventbriteId &&
     !event.bookingEnquiryTeam &&
-    !(event.schedule && event.schedule.length > 1) &&
-    parentEvent.bookingType !== 'Ticketed'
```

Comment:
If this was used as a hack for the previous iteration, could the type be fully removed?

Reply:
yep, I've removed the parent event now